pkg/controller/configuration/configuration.go

```diff
@@ -212,10 +211,9 @@ func (c *Controller) createRevision(config *v1alpha1.Configuration, revName stri
 		}
 		logger.Infof("Created Build:\n%+v", created.Name)
 		c.Recorder.Eventf(config, corev1.EventTypeNormal, "Created", "Created Build %q", created.Name)
-		buildName = created.Name
 	}
 
-	rev := resources.MakeRevision(config, buildName)
+	rev := resources.MakeRevision(config)
```

Comment:
I think this part of the change is a separable distraction from the test change.  Want to send a separate PR?

Reply:
Sure

Reply:
https://github.com/knative/serving/pull/1793